webcam: Add support for mobileNativeCamera option to Webcam and Dashboard (#3844)
transloadit/uppy: website/src/docs/webcam.md

If no preferred image mime type is given, the Webcam plugin will prefer types listed in the [`allowedFileTypes` restriction](/docs/uppy/#restrictions), if any.

### `mobileNativeCamera`

> Replaces Uppy’s custom camera UI on mobile with “Take Picture” and/or “Record Video” buttons that open native device UI for pictures / video (`Function: boolean` || `boolean`, default: `isMobile()`).

Reviewer: Is it really "and/or"? Can you disable one or the other?

Author: Not currently, but since we allow that for the webcam plugin, I thought we need to mimic that functionality here as well.

Author: Added `modes` support, please take a look @Murderlon

> Set to a boolean to forcefully enable / disable this feature, or a function that checks for mobile device and returns a boolean — by default we use  [`is-mobile`](https://github.com/juliangruber/is-mobile) package.

Reviewer: ```suggestion
Set to a boolean to forcefully enable / disable this feature, or a function which returns a boolean. By default we use  the [`is-mobile`](https://github.com/juliangruber/is-mobile) package.
```

> Replaces Uppy’s custom camera UI on mobile with “Take Picture” and / or “Record Video” buttons (depending on [`modes`](#modes) option) that open native device UI for pictures / video (`Function: boolean` || `boolean`, default: `isMobile()`).
> 
> Set to a boolean to forcefully enable / disable this feature, or a function which returns a boolean. By default we use the [`is-mobile`](https://github.com/juliangruber/is-mobile) package.

Reviewer: Some changes to better align the new docs (one sentence + type, clarifications below) and improved grammar. 
```suggestion
Replaces Uppy’s custom camera UI on mobile with the native device camera (`Function: boolean` || `boolean`, default: `isMobile()`).

This will show the “Take Picture” and / or “Record Video” buttons, which ones show depends on the [`modes`](#modes) option.

You can set a boolean to forcefully enable / disable this feature, or a function which returns a boolean. By default we use the [`is-mobile`](https://github.com/juliangruber/is-mobile) package.
```
